routes/index.js

```diff
@@ -0,0 +1,141 @@
+const express = require('express')
+const router = express.Router()
+const db = require('../models')
+
+router.route('/categories')
+.get((req, res) => {
+  db.Filter.find()
+  .then(foundData => res.status(200).json({status: 'success', message: null, data: foundData.map(found => ({id: found._id, name: found.name}))}))
```

Comment:
It could be good idea to prepare some consts for generic statuses and messages - 'success', 'error', 'Already Exist'...